rasa/engine/recipes/config_files/graph_config.yml

```diff
@@ -0,0 +1,532 @@
+# The config recipe.
```

Comment:
Why do we need this file? Is it only for testing?

Reply:
Yes it is used for testing at the moment. We already had default_config.yml here so I've added the graph config next to it. Is it better to move it to data dir?

Reply:
I would move this to a test directory as it's just for testing. The other default config has meaning because it's what we autoconfigure too right?